.env

```diff
@@ -1,3 +1,4 @@
 REACT_APP_APP_VERSION=v2.4.2
 REACT_APP_FEATURE_USE_CUSTOM_VERSION_UPDATE=true
 REACT_APP_FEATURE_DEMO_MODE=false
+REACT_APP_FEATURE_MORE_LANGUAGES=false
```

Comment:
`REACT_APP_FEATURE_BETA_LANGUAGES` looks more suitable as a name for this variable for me